gax-grpc/src/main/java/com/google/api/gax/grpc/ProtoOperationTransformers.java

```diff
@@ -97,12 +86,7 @@ public MetadataT apply(OperationSnapshot operationSnapshot) {
             operationSnapshot.getMetadata() != null ? (Any) operationSnapshot.getMetadata() : null);
       } catch (RuntimeException e) {
         throw ApiExceptionFactory.createException(
-            "Polling operation with name \""
-                + operationSnapshot.getName()
-                + "\" succeeded, but encountered a problem unpacking it.",
-            e,
-            operationSnapshot.getErrorCode(),
-            false);
```

Comment:
You're assuming that the operation failed here, but I think this error scenario is about the unpacking, not about the operation itself failing.